### PR TITLE
[prometheus-blackbox-exporter] - Added option to customize config path by setting it into values file

### PR DESCRIPTION
**Pull request by cristi-vlad**
#### What this PR does / why we need it:

Add possibility to customize config file path

#### Which issue this PR fixes
-

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-blackbox-exporter/values.yaml [hunk 1]
```diff
@@ -102,6 +102,9 @@ config:
         no_follow_redirects: false
         preferred_ip_protocol: "ip4"
 
+# Set custom config path, other than default /config/blackbox.yaml. If let empty, path will be "/config/blackbox.yaml"
+configPath: {}
```
Comment (desaintmartin) [inline charts/prometheus-blackbox-exporter/values.yaml:106]:
```suggestion
# configPath: "/foo/bar"
```

File: charts/prometheus-blackbox-exporter/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 4.11.0
+version: 4.11.1
```
Comment (desaintmartin) [inline charts/prometheus-blackbox-exporter/Chart.yaml:4]:
```suggestion
version: 4.12.0
```

**Review by desaintmartin — CHANGES_REQUESTED** [review]
Thanks! I am curious, what is your use case?

**Comment by cristi-vlad** [comment]
> Thanks! I am curious, what is your use case?

Hello. Thanks for fast update.
Our case is integration with vault & vault-agent-injector.

Thank you,
/Cristi

**Pull request closed without merging**